shipyard_blueprints/shipyard-bp-utils/shipyard_bp_utils/files.py

```diff
@@ -231,12 +232,33 @@ def find_all_local_file_names(source_folder_name: str = None) -> list:
     list: A list of all files that exist in the current working directory.
     """
     logger.debug(f"Finding all local file names in {source_folder_name}...")
-    cwd = os.getcwd()
-    cwd_extension = os.path.normpath(f"{cwd}/{source_folder_name}/**")
-    all_paths = glob.glob(cwd_extension, recursive=True)
-    logger.debug(f"Found {len(all_paths)}.")
-    logger.debug(all_paths)
-    return remove_directories_from_path_list(all_paths)
+    # cwd = os.getcwd()
+    # if not source_folder_name:
+    #     search_path = os.getcwd()
+    # else:
+    #     search_path = os.path.normpath(f"{cwd}/{source_folder_name}/**")
+    # all_paths = glob.glob(search_path, recursive=True)
+    # print(all_paths)
+    # logger.debug(f"Found {len(all_paths)}.")
+    # logger.debug(all_paths)
+    # return remove_directories_from_path_list(all_paths)
+    if source_folder_name is None:
```

Comment:
needs to handle `""` and `None` I would simplify this to something like source_folder_name = source_folder_name or os.getcwd()
or if not source_folder_name:
     source_folder_name = os.getcwd()